ov/profile_templates.go

```diff
@@ -241,7 +243,15 @@ func SetMp(mp ManagementProcessors) IntManagementProcessor {
 						arg[strings.ToLower(string(typeOfS.Field(j).Name[0]))+typeOfS.Field(j).Name[1:]] = v.Field(j).Interface()
 					}
 				}
-				ags = append(ags, arg)
+				//Check generation of server
+				gen := sht[6:11]
```

Comment:
rather than doing string manipulation to get the generation, OV already provides a field called generation. Use that instead

Reply:
Done